### PR TITLE
[7.59.x][JBPM-10012] - Right Springboot version should be 2.3.10

### PR DESCRIPTION
**Pull request by afalhambra**
**[JBPM-10012](https://issues.redhat.com/browse/JBPM-10012)**: Right Springboot version should be 2.3.10

**referenced Pull Requests**: 

* https://github.com/kiegroup/droolsjbpm-integration/pull/2711

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest</b> (using <i>this</i> e.g. <b>Jenkins retest this</b> optional but no longer required)
 
* for a <b>full downstream build</b> 
  * for <b>jenkins</b> job: please add comment: <b>Jenkins run fdb</b>
  * for <b>github actions</b> job: add the label `run_fdb`
    
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -263,7 +263,7 @@
     <version.org.sonatype.sisu>2.3.0</version.org.sonatype.sisu>
     <version.org.sonatype.sisu.sisu-guice>3.2.3</version.org.sonatype.sisu.sisu-guice>
     <version.org.springframework>5.2.19.RELEASE</version.org.springframework>
-    <version.org.springframework.boot>2.3.4.RELEASE</version.org.springframework.boot>
+    <version.org.springframework.boot>2.3.10.RELEASE</version.org.springframework.boot>
     <version.org.springframework.amqpboot>2.3.4</version.org.springframework.amqpboot>
```
Comment (gmunozfe) [inline pom.xml:267]:
```suggestion
    <version.org.springframework.amqpboot>2.3.10</version.org.springframework.amqpboot>
```

**Review by gmunozfe — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (afalhambra) [reply pom.xml]: Done. Thanks!

**Comment by mareknovotny** [comment]
jenkins do fdb

**Review by mareknovotny — APPROVED** [review]

**Review by gmunozfe — APPROVED** [review]
Looks good to me, well done @afalhambra !
As there's no 2.3.10 amqp-10-jms-spring-boot-starter, it seems correct to upgrade to the nearest 2.3.9.

**Comment by afalhambra** [comment]
Failing test is not related

**Comment by afalhambra** [comment]
@mareknovotny this should be ready to merge since failing test is not related

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
waiting on FDB as the previous run didn't complete even with 1 failure not related ;) @afalhambra

**Comment by afalhambra** [comment]
Same error again in the same test `org.kie.dmn.feel.client.showcase.FEELShowcaseIT.org.kie.dmn.feel.client.showcase.FEELShowcaseIT`

```
Error Message
io.github.bonigarcia.wdm.WebDriverManagerException: Error HTTP 404 executing GET http://npm.taobao.org/mirrors/geckodriver HTTP/1.1

Stacktrace
io.github.bonigarcia.wdm.WebDriverManagerException: io.github.bonigarcia.wdm.WebDriverManagerException: Error HTTP 404 executing GET http://npm.taobao.org/mirrors/geckodriver HTTP/1.1
	at org.kie.dmn.feel.client.showcase.FEELShowcaseIT.setupClass(FEELShowcaseIT.java:49)
Caused by: io.github.bonigarcia.wdm.WebDriverManagerException: Error HTTP 404 executing GET http://npm.taobao.org/mirrors/geckodriver HTTP/1.1
	at org.kie.dmn.feel.client.showcase.FEELShowcaseIT.setupClass(FEELShowcaseIT.java:49)
```

Trying once more
Jenkins run fdb

**Comment by afalhambra** [comment]
> FEELShowcaseIT

Running it once more, though I can see that even if that test got failed, the build keeps on going till it gets completed - am I wrong?

**Comment by mareknovotny** [comment]
now it is ok, as the build is yellow on jenkins, so everything ran until the end ;).

**Pull request merged**